test/karma.conf.js

```diff
@@ -0,0 +1,84 @@
+// Karma configuration
+// Generated on Mon Jan 18 2016 19:06:38 GMT+0100 (CET)
+
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '../',
+
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha', 'expect', 'sinon'],
+
+
+    // list of files / patterns to load in the browser
+    files: [
+      'http://openlayers.org/en/master/build/ol.js',
+      'http://cdnjs.cloudflare.com/ajax/libs/extjs/6.0.0/ext-all.js',
+      'test/loader.js',
+      'src/**/*.js',
+      'test/spec/GeoExt/**/*.test.js'
+    ],
```

Comment:
Hasn't been an issue with existing tests but may be useful in the future. Anyway, I would probably keep them out until they are actually used to reduce load times?